xformers/components/attention/csrc/cuda/mem_eff_attention/attention_forward_generic.cu

```diff
@@ -143,11 +143,12 @@ efficient_attention_forward_generic(
                       value.stride(1) % Kernel::kAlignmentV == 0,
                       "value is not correctly aligned");
 
-                  res = at::zeros(
+                  res = at::empty(
```

Comment:
The zeros was not needed before because we handle the kernel differently according if it's the first iteration or not (write vs accumulate)